Resources/Prototypes/_Goobstation/Catalog/Fills/Belt/belts.yml

```
@@ -2,7 +2,7 @@
   parent: [ ClothingBeltMilitaryWebbingCMO, BaseMedicalContraband ]
   suffix: Filled
   id: ClothingBeltMilitaryWebbingCMOFilled
-  name: chief medical officer's chest rig
+  name: medical LBV-330 # mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)